tests/parser/parser.cpp

```diff
@@ -37,11 +37,11 @@ TEST(Parser, invalidXMLElements)
     const std::vector<std::string> expectedErrors = {
         "Specification mandate value for attribute bearded.",
         "Specification mandates value for attribute bearded.",
-        "Opening and ending tag mismatch: Dwarf line 3 and ShortGuy.",
-        "Opening and ending tag mismatch: Hobbit line 4 and EvenShorterGuy.",
-        "Opening and ending tag mismatch: Wizard line 5 and SomeGuyWithAStaff.",
-        "Opening and ending tag mismatch: Elf line 6 and fellows.",
-        "Premature end of data in tag fellowship line 2.",
+        "LibXml2 error: Opening and ending tag mismatch: Dwarf line 3 and ShortGuy.",
```

Comment:
I like the starting tags!  Makes it a bit more contextual when you're trying to interpret the error :)